src/api/firebase.js

```diff
@@ -74,12 +96,13 @@ export async function addItem(listId, { itemName, daysUntilNextPurchase }) {
 	});
 }
 
-export async function updateItem() {
-	/**
-	 * TODO: Fill this out so that it uses the correct Firestore function
-	 * to update an existing item. You'll need to figure out what arguments
-	 * this function must accept!
-	 */
+export async function updateItem(listId, listItemId) {
```

Comment:
Same comment as above about adding comments to this function. Notice how the other API-related functions in this file are setup - listing params with brief descriptions plus a short summary of what the function actually does. Very useful for maintaining clarity as you iterate and add to this file.

Reply:
Thanks for the reminder DJ! Going to work on implementing comments more often as I know how important and useful they are.